test/cloudops.go

```diff
@@ -156,6 +148,17 @@ func compute(t *testing.T, driver cloudops.Ops) {
 		require.NoErrorf(t, err, fmt.Sprintf("error occured while getting cluster size after being set with 0 timeout. Error:[%v]", err))
 	}
 
+	instanceToDelete := os.Getenv("INSTANCE_TO_DELETE")
+	zoneOfInstanceToDelete := os.Getenv("INSTANCE_TO_DELETE_ZONE")
+	if instanceToDelete != "" {
+		err := driver.DeleteInstance(instanceToDelete, zoneOfInstanceToDelete, 5*time.Minute)
+		if _, ok := err.(*cloudops.ErrNotSupported); ok {
+			return
+		}
+		require.NoError(t, err, fmt.Sprintf("failed to delete instance [%v]. Error:[%v]", instanceToDelete, err))
+	} else {
+		logrus.Fatalf("Set INSTANCE_TO_DELETE environment variable")
```

Comment:
looks like the test user has to create a dedicated vm to get deleted before running the test. This should be optional and should not fail other tests.

Reply:
actually looks like this was already existing. Just make the variable optional as part of this change.